README.md

```diff
@@ -26,14 +26,15 @@ import smartcar "github.com/smartcar/go-sdk"
 	smartcarClient := smartcar.NewClient()
 	```
 
-1. Initialize an `Auth` Client struct with your `client id`, `client secret`, `redirect URI`, the `scopes` you want, and `test mode`.
+1. Initialize an `Auth` Client struct with your `client id`, `client secret`, `redirect URI`, the `scopes` you want, [`Country`](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2), and `test mode`.
 
 	```go
 	// An Auth Client is used to generate a smartcar connect url, authenticate with smartcar, and check compatibility
 	authClient := smartcarClient.NewAuth(&smartcar.AuthParams{
 		ClientID:     "<CLIENT_ID>",
 		ClientSecret: "<CLIENT_SECRET>",
 		RedirectURI:  "<REDIRECT_URI>",
+                Country:      "<Country Code>",
```

Comment:
@allisonc07 @rsimari 
Sorry for the delayed comment but I also looked over ruby SDK i have a question. There is a difference in what we pass during the auth. Here we expect developers to pass something like `flags: 'country:US'` where as in go SDK we just added an additional parameter `country`. I am assuming you guys did the other SDKs as well, so is it like different for different SDKs ? I think t is worth it to keep th signature somewhat similar, WDYT ?

Reply:
I was unsure on this as well, my original assumption was that we'd do something like `flags: 'country:US'` so that way we wouldn't need to do a breaking change to the SDKs when decide to promote `country` from being a flag to a top level parameter.

Reply:
For the other SDKs (api + connect) that I worked on, I have been expecting the `flags` parameter as a space separated list. I thought that that might be the best approach since it would require the least amount of maintenance if we were to add new flags. 

In terms of defaulting, I think we have decided to default to 'US' in the other SDKs, so we should change that here as well!